aspnetcore/migration/webapi.md

```diff
@@ -4,19 +4,187 @@ author: ardalis
 description: Learn how to migrate a web API implementation from ASP.NET 4.x Web API to ASP.NET Core MVC.
 ms.author: scaddie
 ms.custom: mvc
-ms.date: 05/26/2020
+ms.date: 01/31/2022
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: migration/webapi
 ---
 # Migrate from ASP.NET Web API to ASP.NET Core
 
-By [Scott Addie](https://twitter.com/scott_addie) and [Steve Smith](https://ardalis.com/)
-
 An ASP.NET 4.x Web API is an HTTP service that reaches a broad range of clients, including browsers and mobile devices. ASP.NET Core combines ASP.NET 4.x's MVC and Web API app models into a single programming model known as ASP.NET Core MVC. This article demonstrates the steps required to migrate from ASP.NET 4.x Web API to ASP.NET Core MVC.
 
 [View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/migration/webapi/sample) ([how to download](xref:index#how-to-download-a-sample))
 
-::: moniker range=">= aspnetcore-3.0"
+::: moniker range=">= aspnetcore-6.0"
+
+## Prerequisites
+
+[!INCLUDE [prerequisites](../includes/net-core-prereqs-vs-3.1.md)]
+
+## Review ASP.NET 4.x Web API project
+
+This article uses the *ProductsApp* project created in [Getting Started with ASP.NET Web API 2](/aspnet/web-api/overview/getting-started-with-aspnet-web-api/tutorial-your-first-web-api). In that project, a basic ASP.NET 4.x Web API project is configured as follows.
```

Comment:
"No need to have them bring that".    What do you mean by "that".  No need to bring in the previous project?  Sorry could you be more explicit?

Reply:
Don't bring in the old project. They just need the model and controller.

Reply:
Sorry, one more question to make sure I am on the same page.  So just keep to migrating the controller, the model it uses and program.cs to show where routing is enabled.  I also pull out the section: _Review ASP.NET 4.x Web API project_ which includes mention of the original WebAPIConfig.cs attribute routing.  Do you want to compare old routing to new routing or just stick with how the new routing is set up?  Thanks for all the help.

Reply:
Correct. No need to review the project, those folks are experts in 4.x web api.
No need to compare old routing to new routing, provide links to routing and creating API content.

Reply:
OK, it sounds like I will just instruct to create a new ASP.NET Core Web API project rather than the blank solution first.  I am not to show the old controller or model (nor the whole previous project), but rather just show the aspects of what a new controller and model looks like and what is happening with that.    That feels really narrow compared to the title.  Should I change the title to be specific to migrating a 4.x controller rather than a 4.x Web API project?   Sorry for all the questions.